functions.js channel not initialized on international YT

raszpl opened the pull request:
Switched language and 'ytd-c4-tabbed-header-renderer' became 'yt-subscribe-button-view-model-wiz' somehow

ImprovedTube commented:
hi! @raszpl thanks, are we including yt-subscribe-button-view-model-wiz then? 


 we can hoard all DOM names.  
 (and eventually one could collect statistics how common they are or not)

ImprovedTube commented:
hi! @raszpl thanks, are we including yt-subscribe-button-view-model-wiz then? 


 we can hoard all DOM names.  
 (and eventually one could collect statistics how common they are or not)
 
---
 
Happy to report BTW that everything merged is live! At https://chromewebstore.google.com/detail/improve-youtube-%F0%9F%8E%A7-for-yo/bnomihfieiccainjcjblhegjgglakjdd -  while [uninstalls](https://improvedtube.com/uninstalls-hours-days-usatime-updated-every-3-minutes) arent going up

ImprovedTube commented:
hi! @raszpl thanks, are we including yt-subscribe-button-view-model-wiz then? 


 we can hoard all DOM names.  
 (and eventually one could collect statistics how common they are or not)
 
---
 
Happy to report BTW, that everything merged is live! At https://chromewebstore.google.com/detail/improve-youtube-%F0%9F%8E%A7-for-yo/bnomihfieiccainjcjblhegjgglakjdd -  while [uninstalls](https://improvedtube.com/uninstalls-hours-days-usatime-updated-every-3-minutes) arent going up

ImprovedTube commented:
hi! @raszpl thanks, are we including yt-subscribe-button-view-model-wiz then? 


 we can hoard all DOM names.  
 (and eventually one could collect statistics how common they are or not)
 
---

Happy / proud to tell BTW, that everything merged is live once again! At https://chromewebstore.google.com/detail/improve-youtube-%F0%9F%8E%A7-for-yo/bnomihfieiccainjcjblhegjgglakjdd,  while [uninstalls / reinstalls](https://improvedtube.com/uninstalls-hours-days-usatime-updated-every-3-minutes) aren't raised

raszpl commented:
> hi! @raszpl thanks, are we including yt-subscribe-button-view-model-wiz then?

thankfully it happened on 

`<yt-subscribe-button-view-model class="yt-subscribe-button-view-model-wiz" style="width: 172px;">`

so no need
> we can hoard all DOM names. (and eventually one could collect statistics how common they are or not)

> Happy / proud to tell BTW, that everything merged is live once again! At https://chromewebstore.google.com/detail/improve-youtube-%F0%9F%8E%A7-for-yo/bnomihfieiccainjcjblhegjgglakjdd, 

👍 

>while [uninstalls / reinstalls]

people probably havent woken up yet :]

>(https://improvedtube.com/uninstalls-hours-days-usatime-updated-every-3-minutes) aren't raised

a ton of those from 1-2 weeks ago was probably mine until I removed last line from background.js in my local debug copy :P

ImprovedTube commented:
> a ton of those from 1-2 weeks ago was probably mine until I removed last line from background.js in my local debug copy :P

how?  1000 is little since we are featured at  firefox about:addons ( https://github.com/code-charity/youtube/discussions/2411 )

raszpl commented:
> > a ton of those from 1-2 weeks ago was probably mine until I removed last line from background.js in my local debug copy :P
> 
> how? 

I had to install/uninstall fo debugging and was triggering the report, got annoyed after few days and removed it from local copy

ImprovedTube commented:
💡now i remembered whats the last line! 
reloading should be enough.

raszpl commented:
> 💡now i remembered whats the last line! reloading should be enough.

not when one is testing "onInstall" code while lazy :-)
